src/EnergyPlus/WindowManagerExteriorOptical.cc

```diff
@@ -388,7 +388,7 @@ namespace Window {
     {
```

Comment:
I have two large PRs that should go in this week or next (FLW) and that will have a lot of conflicts with these changes. You may want to hold off on these until those are in.